<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowDialog</class>
 <widget class="QDialog" name="WindowDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置窗位与窗宽</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1" colspan="3">
    <widget class="QSpinBox" name="widthEdit">
     <property name="minimum">
      <number>-2000</number>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QSpinBox" name="levelEdit">
     <property name="minimum">
      <number>-2000</number>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QComboBox" name="presetComboBox"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_1">
     <property name="text">
      <string>使用预设：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>窗宽：</string>
     </property>
     <property name="buddy">
      <cstring>widthEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>窗位：</string>
     </property>
     <property name="buddy">
      <cstring>levelEdit</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>WindowDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>WindowDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>presetComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>WindowDialog</receiver>
   <slot>on_selectionChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
